### PR TITLE
ciscosmb tenant

### PR DESCRIPTION
**Pull request by Andersson007**

### CODE REVIEW

**Comment by Andersson007** [comment]
https://github.com/ansible-collections/community.ciscosmb/issues/25#issuecomment-853000054

File: zuul/tenants.yaml [hunk 1]
```diff
@@ -39,6 +39,7 @@
           - ansible-collections/cloud.common
           - ansible-collections/community.asa
           - ansible-collections/community.aws
+          - ansible-collections/community.ciscosmb
```
Comment (pabelanger) [inline zuul/tenants.yaml:42]: will you be gating these project with zuul? EG: zuul merging, or just want access to publish jobs?

Comment (pabelanger) [reply zuul/tenants.yaml:42]: if just publish jobs, please move to line 106 an below

Comment (gundalow) [reply zuul/tenants.yaml:42]: Just publish

Comment (Andersson007) [reply zuul/tenants.yaml]: Done

**Pull request merged**